exchange/gdpr_test.go

```diff
@@ -148,6 +148,65 @@ func TestGetConsent(t *testing.T) {
 	}
 }
 
+func TestSelectEEACountries(t *testing.T) {
+	tests := []struct {
+		description         string
+		hostEEACountries    []string
+		accountEEACountries []string
+		expected            []string
+	}{
+		{
+			description:         "Account EEA countries provided, return account's",
```

Comment:
Nitpick: Use underscores instead of spaces when separating words in test name so `t.Run` doesn't perform space for underscore substitutions making it easier to find failing tests in the code base via search.